src/test/java/com/unit/UNIT/TEST/MachineTest.java

```diff
@@ -47,7 +50,63 @@ public void shouldIncreaseAmountOfMoney2(){
         verify(validator,times(1)).validateMoney(money);
 
     }
-*/
 
+    @Test
+    public void shouldClearAmountOfMoney() {
+        //given
+        int finalResult = 0;
+        //when
+        machine.winInfo();
+        //then
+        assertEquals(machine.bankAmount, finalResult);
+    }
+
+    @Test
+    public void shouldFiveElementsInList() {
```

Comment:
should what? Postaraj się pisać nazwy testów tak, jakby to były zdania, np. "Powinno wygenerować pięć znaków"